### PR TITLE
Splitted value and unit of the result, made response delay configurable

### PR DESCRIPTION
**Pull request by anti43**
Splitted value and unit of the result, made response delay configurable, typo

### CODE REVIEW

File: src/main/java/pt/lighthouselabs/obd/commands/ObdCommand.java [hunk 1]
```diff
@@ -28,6 +28,7 @@
   protected String cmd = null;
   protected boolean useImperialUnits = false;
   protected String rawData = null;
+  protected long responseTimeDelay = 200;//automatic sampling rate detection coming
```
Comment (pires) [inline src/main/java/pt/lighthouselabs/obd/commands/ObdCommand.java]: Can you make this `private static final` please?

Comment (anti43) [inline src/main/java/pt/lighthouselabs/obd/commands/ObdCommand.java]: I plan to measure this for each command and set it accordingly, so neither final nor static apply..

Comment (pires) [inline src/main/java/pt/lighthouselabs/obd/commands/ObdCommand.java]: Then make it `private final` and set it in a different constructor. In default constructor, use `200`.

Comment (anti43) [inline src/main/java/pt/lighthouselabs/obd/commands/ObdCommand.java]: ok

**Comment by pires** [comment]
It will need rebase.

**Comment by pires** [comment]
#38 or whatever comes next seems to close this.

**Pull request closed without merging**